index.html

```diff
@@ -1,19 +1,105 @@
-<!doctype html>
+<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN"
+"http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd">
 
-<html lang="en">
-<head>
-  <meta charset="utf-8">
-
-  <title>My Resume</title>
+<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en">
 
-  <link rel="stylesheet" href="css/index.css">
+<head>
+    <link href="https://fonts.googleapis.com/css?family=Titillium+Web" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css?family=Work+Sans" rel="stylesheet">
+    <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
+     <link rel = "stylesheet"
+        type = "text/css"
+        href = "C:\Git\Preprocessing-I\css\index.css" />
+     <title>Jarred Stanford's Resume</title>
+     <meta name="viewport" content="width=device-width, initial-scale=1.0">
 
-  <!--[if lt IE 9]>
-    <script src="https://cdnjs.cloudflare.com/ajax/libs/html5shiv/3.7.3/html5shiv.js"></script>
-  <![endif]-->
 </head>
 
 <body>
-  <h1>My Resume</h1>
+    <header>
+        <nav>
+            <a href="">Home</a>
+            <a href="jarredstanford.github.io">Portfolio</a>
```

Comment:
```suggestion
            <a href="http://jarredstanford.github.io">Portfolio</a>
```
Needs the http or else it tries to find jarredstandford.github.io in your file structure